distribution/openrct2.d.ts

```diff
@@ -2711,6 +2711,11 @@ declare global {
          * on the direction its moving in.
          */
         travelBy(distance: number): void;
+
+        /**
+         * Moves the vehicle to the track piece specified in the parameters.
+         */
+        moveToTrack(coord: CoordsXY, elemIndex: number): void;
```

Comment:
```suggestion
        moveToTrack(x: number, y: number, trackElementIndex: number): void;
```
And also have `x` and `y` be tile coordinates instead of entity coordinates would probably be neater. 🙂

Reply:
@guysv Can you still implement this suggestion?